### PR TITLE
Adding GSoC'22 Final Report

### PR DESCRIPTION
**Pull request by ganimtron-10**
Adding final report for the GSoC'22.

It is also visible in the blog.html

image

### CODE REVIEW

**Comment by skoudoro** [comment]
Thank you for being so fast @ganimtron-10!

I will generate the doc tomorrow. check it and merge it all good.

File: docs/source/posts/2022/2023-01-24-final-report-praneeth.rst [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+   ii. **Mode Panel** 
+        It is a composite UI element consisting of the main panel(``Panel2D``) on which buttons(``Button2D``) are arranged which can toggle the current working mode. Each button has an icon associated with it which tries to depict the information about the mode. Here mode is nothing but the different channels which on selection can perform different tasks. Some of the modes present in the Mode Panel are discussed below:
+            -  Selection:  This mode is used to select an individual or group of shapes.
+            -  Deletion:  This mode is used to delete an individual or group of shapes.
+            -  The modes mentioned below create an element on the Panel which is described below.
+                -  Line: 
+                -  Quad: 
+                -  Circle: 
+                -  Polyline: 
+                -  Freehand drawing:
```
Comment (skoudoro) [inline docs/source/posts/2022/2023-01-24-final-report-praneeth.rst:63]: why there are `:` at the end of each line?

**Review by skoudoro — COMMENTED** [review]
Nice!  Thank you @ganimtron-10.

Can you address the small comment below and then, I will go ahead and merge this PR. 

Thank you!

File: docs/source/posts/2022/2023-01-24-final-report-praneeth.rst (hunk 1 above)
Comment (ganimtron-10) [reply docs/source/posts/2022/2023-01-24-final-report-praneeth.rst]: In my gist, I added the shape icons for each but as this is a blog the temporary icons may not be available in future so removed it but forgot to remove the colons.
I have updated the blog.

**Comment by skoudoro** [comment]
Thank you for the update, the report look nice, merging

**Pull request merged**